frontend/src/components/TheFooter.vue

```diff
@@ -1,23 +1,20 @@
 <template>
   <v-footer v-if="!$route.meta.hideNavigation" app color="navbar" class="d-flex flex-column">
-    <div class="d-flex w-100 align-center position-relative">
-      <div class="text-center w-100">
-        <p class="footerdescription">The work is supported by the National Science Foundation (NSF) under award 2322510: “Synthesizing hydrologic process knowledge through a community database.”</p>
-       <div class="footeryearmark">{{ new Date().getFullYear() }} — <strong>hydroprocess_db</strong></div> 
+    <div class="d-flex w-100 position-relative">
+        <div class="nsfimagecontainer float-left">
+          <img :src="nfsImgUrl" cover  width="100" height="50" class="rounded">
```

Comment:
@MSDrao my only comment here is that the NSF image is out of proportion -- it appears "squished"

![image](https://github.com/user-attachments/assets/c7a0ae5b-c825-48b4-a8d2-4e232bf80751)

Reply:
I'm going to merge in this PR and if you have time, you can fix the proportions on the NSF image @MSDrao